etc/workflows/publish-uploaded-assets.xml

```diff
@@ -49,7 +49,8 @@
       exception-handler-workflow="partial-error"
       description="Update recording in Opencast Media Module">
       <configurations>
-        <configuration key="download-source-flavors">${download-source-flavors}</configuration>
+        <configuration key="download-source-flavors">${download-source-flavors},dublincore/*,security/*</configuration>
```

Comment:
Updating already published metadata on asset upload isn't intuitive. Here you only want to add new asset to the publication.